src/mongoModelsRepo/getAttributes.ts

```diff
@@ -1,10 +1,10 @@
-import Attribute from '../models/Attribute';
-import GetAttributesOptions from '../repoFactory/options/GetAttributesOptions';
-import GetAttributesResult from '../repoFactory/results/GetAttributesResult';
-import Config from './Config';
+import type Attribute from '../models/Attribute';
+import type GetAttributesOptions from '../repoFactory/options/GetAttributesOptions';
+import type GetAttributesResult from '../repoFactory/results/GetAttributesResult';
+import type Config from './Config';
 import pagination from './utils/pagination';
 
 export default (config: Config) => {
   return async (opts: GetAttributesOptions): Promise<GetAttributesResult> =>
-    pagination(config, 'personaAttributes')<Attribute>(opts);
+    await pagination(config, 'personaAttributes')<Attribute>(opts);
```

Comment:
Is this a linting rule thing or just a preference? Shouldn't make a difference right?

Reply:
@ToranSharma , it's a linting rule, yeah. I don't think it's going to make a difference 🤔